init.lua

```diff
@@ -330,6 +336,12 @@ function PaperWM:start()
     window_list = {}
     index_table = {}
     ui_watchers = {}
+    is_floating = {}
+
+    local persisted = hs.settings.get(IsFloatingKey) or {}
+    for _, id in ipairs(persisted) do
+        is_floating[id] = true
```

Comment:
Should we check if this window id is still valid before adding to `is_floating` list?

Reply:
Fixed. I also added a check to ensure we don't add windows that are rejected by the current filter.